jc/parsers/amixer.py

```diff
@@ -0,0 +1,261 @@
+r"""jc - JSON Convert `amixer sget` command output parser
+Usage (cli):
+    $ amixer sget <control_name> | jc --amixer
+    $ amixer sget Master | jc --amixer
+    $ amixer sget Capture | jc --amixer
+    $ amixer sget Speakers | jc --amixer
+Usage (module):
+    import jc
+    result = jc.parse('amixer', <amixer sget command output>)
+Schema:
+{
+    "control_name":                     string,
+    "capabilities": [
+                                        string
+    ],
+    "playback_channels": [
+        string
+    ],
+    "limits": {
+        "playback_min":                 string,
+        "playback_max":                 string
+    },
+    "mono": {
+        "playback_value":               string,
+        "percentage":                   string,
+        "dB":                           string,
+        "status":                       string
+    }
```

Comment:
These should be converted to floats and ints in the `_process` function.

Reply:
@kellyjonbrazil I have 2 questions:
1. So the `"raw"` option will always act as string?
2. The Scheme is representation of the `_process` output? (When `raw=False`)?

Reply:
That is correct.